### PR TITLE
fix 745: remove self attstn from U2F attstn format

### LINKED ISSUE #745: U2F Attestation Statement Format does not define "self attestation" steps
The U2F Attestation Statement Format does not define steps for performing "self attestation", which e.g. [is done in the packed attestation stmt format section](https://w3c.github.io/webauthn/#ref-for-self-attestation%E2%91%A0%E2%91%A1).

### PR DESCRIPTION
**Pull request by equalsJeffH**
fixes #745, related to PR #746 (merge latter before this one)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/781.html" title="Last updated on Feb 7, 2018, 4:18 PM GMT (d62e066)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/781/d56d1e7...d62e066.html" title="Last updated on Feb 7, 2018, 4:18 PM GMT (d62e066)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -3365,7 +3365,7 @@ This attestation statement format is used with FIDO U2F authenticators using the
 :: fido-u2f
 
 : Attestation types supported
-:: [=Basic Attestation=], [=Self Attestation=], [=Attestation CA=]
+:: [=Basic=], [=AttCA=]
```
Comment (agl) [inline index.bs]: Is `AttCA` a typo? (I see other references to `[=Attestation CA=]` in the file, but none to `AttCA`.)

Comment (agl) [reply index.bs]: Oh, perhaps it works in light of [this change](https://github.com/w3c/webauthn/pull/780/commits/a9f806ce651edddefb1eec3c8d539752c9b001eb).

Comment (equalsJeffH) [reply index.bs]: yeah that's why i suggest merging pr #746 before this one, in the OP https://github.com/w3c/webauthn/pull/781#issue-294940252

**Comment by nadalin** [comment]
@emlun Please review

**Comment by nadalin** [comment]
@equalsJeffH Can you merge ?

**Pull request merged**